README.md

```diff
@@ -188,113 +252,166 @@ The following CSS:
+```css
+.wrapper {
+	width: calc(100% - 2em);
+	margin-left: 1em;
+	margin-right: 1em;
+}
+```
 
 
 # Why?
 
 This plugin was spawned out of a [discussion on the `cssnext` repo](https://github.com/cssnext/cssnext/issues/99) and a personal need.
 
-There is another similar plugin available, [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), which at the moment, covers less of the [CSS-variable/custom-property draft/spec](http://dev.w3.org/csswg/css-variables/) compared to this plugin. It also happened to not cover the feature I was after and after dealing with it, making an incomplete prototype piggybacking off of it; I decided to write my own starting from scratch. I do use some of the same/similar unit tests because that just makes sense to get proper code coverage.
+There is another similar plugin available, [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), although it restricts itself much more than this, preferring spec conformance over imperfect feature support.
+
+### Interoperability
+
+If you are/were already using [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), this should work out of the box, without issue, and other than differences in the JavaScript API, it's a drop-in replacement. The only difference in CSS handling is that this attempts much broader support, and you can just start taking advantage immediately. Note the [caveats](#caveats), in that although this does correctly support what `postcss-custom-properties` does, there are certain edge cases it doesn't get perfectly.
 
 ### Differences from `postcss-custom-properties`
 
-The main features that we`postcss-css-variables` add/provide are:
+In [`postcss-custom-properties`](https://github.com/postcss/postcss-custom-properties), CSS variable declarations are specifically restricted to the `:root` selector. In `postcss-css-variables`, this is not the case and they may be declared inside any rule with whatever selector.
+ 
 
- - No limitation on what scope CSS variables can be declared or used (`:root` or wherever)
- 	 - Proper value substitution based on explicit DOM/structure traversal
- - At-rule support `@media`, `@support`, etc
- - Nested rules which can be fully deduced with [`postcss-nested`](https://github.com/postcss/postcss-nested).
- - Pseudo class/element support `:hover`, etc
+Here's a quick overview of the differences:
 
+ - CSS variables may be declared in any selector like `.foo` or `.foo .bar:hover`, and is not limited to just `:root`
+ - CSS variables may be declared in `@media`, `@support`, and other at-rules.
+ - CSS variables may be declared in `:hover` and other psuedo-classes, and they are evaluated properly.
```

Comment:
That's inconsistent with the rest of the list, which are all complete sentences. I don't want to sound too much like a grammar nazi, but you get what I mean.